### PR TITLE
Update CUDA base images

### PR DESCRIPTION
**Pull request by mattt**
Related to #1162

Nvidia recently unpublished some tagged images from Docker Hub, which is breaking cog build for some users.

This PR regenerates the list of available CUDA images to ensure that missing images like `11.2.1-cudnn8-devel-ubuntu20.04` aren't requested.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -28,6 +28,10 @@ pkg/dockerfile/embed/cog.whl: python/* python/cog/* python/cog/server/* python/c
 	mkdir -p pkg/dockerfile/embed
 	cp dist/*.whl $@
 
+.PHONY: compatibility-matrices
+compatibility-matrices:
+	$(GO) run tools/generate_compatibility_matrices/main.go
+
```
Comment (bfirsh) [inline Makefile:34]: `make generate` does this I think?

**Review by bfirsh — APPROVED** [review]
Ah, nvidia, this has broken a bunch of times before.

I suppose this means CUDA 8-10 is not supported in Cog any longer? We should probably note that in the release notes.

**Comment by zeke** [comment]
> Nvidia recently unpublished some tagged images from Docker Hub

Do we know why?

File: Makefile (hunk 1 above)
Comment (mattt) [reply Makefile:34]: Good callout. Rebased out of #1170

**Comment by mattt** [comment]
Superseded by #1170

**Pull request closed without merging**